lightstep/resource_metric_condition_test.go

```diff
@@ -48,8 +48,6 @@ resource "lightstep_metric_condition" "errors" {
   project_name = "terraform-provider-tests"
   name = "Too many requests"
   expression {
-	  evaluation_window   = "2m"
-	  evaluation_criteria = "on_average"
```

Comment:
this was missed a while back, these fields no longer exist

Reply:
🙏